tests/unit/remote/gdrive/conftest.py

```diff
@@ -0,0 +1,140 @@
+from datetime import datetime, timedelta
+import json
+import mock
+
+from google_auth_oauthlib.flow import InstalledAppFlow
+import google.oauth2.credentials
+
+import pytest
+
+from dvc.repo import Repo
+from dvc.remote.gdrive import RemoteGDrive
+from dvc.remote.gdrive.client import GDriveClient
+from dvc.remote.gdrive.utils import MIME_GOOGLE_APPS_FOLDER
+from dvc.remote.gdrive.oauth2 import OAuth2
+
+
+AUTHORIZATION = {"authorization": "Bearer MOCK_token"}
+FOLDER = {"mimeType": MIME_GOOGLE_APPS_FOLDER}
+FILE = {"mimeType": "not-a-folder"}
+
+COMMON_KWARGS = {
+    "data": None,
+    "headers": AUTHORIZATION,
+    "timeout": GDriveClient.TIMEOUT,
+}
+
+
+class Response:
+    def __init__(self, data, status_code=200):
+        self._data = data
+        self.text = json.dumps(data) if isinstance(data, dict) else data
+        self.status_code = status_code
+
+    def json(self):
+        return self._data
+
+
+@pytest.fixture()
+def repo():
```

Comment:
Do we need to use repo from the dvc code repo itself? Or can we use `dvc_repo` fixture, as we do everywhere else?

Reply:
This unit tests doesn't need the real repo, so it is a bit excessive to setup/teardown the repo for each test. But it may be a good idea to create one temporary just for them all. Would it be ok to make this fixture `scope="module"` and take/return `dvc_repo`?

Reply:
Sounds good :)